packages/core/src/index.ts

```diff
@@ -43,6 +43,11 @@ interface PersistentStorage {
   set: (key: string, value: string) => void;
 }
 
+type flagCacheConfig = {
+  refetchFlags: boolean,
```

Comment:
the type suggests already that its all about flags? =)
otherwise it would be `timeToLiveFlags` too? :shrug: 

```suggestion
  refetch: boolean,
```